module/network/RobotCommunication/data/config/RobotCommunication.yaml

```diff
@@ -0,0 +1,4 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+port: // find port of virtual season. 4 digit
```

Comment:
http://humanoid.robocup.org/wp-content/uploads/v-hsc_server_specification_v1.03.pdf
port is 3737